gradlew.bat

```diff
@@ -14,7 +14,7 @@
 @rem limitations under the License.
 @rem
 
-@if "%DEBUG%" == "" @echo off
+@if "%DEBUG%"=="" @echo off
```

Comment:
The changes in `gradlew.bat` were created by using `./gradlew wrapper --gradle-version=7.6.2`